generators/app/aspnetHelper.js

```diff
@@ -71,6 +71,9 @@ AspNetHelper.prototype.createDockerComposeFile = function (isDebug) {
  * @returns {string}
  */
 AspNetHelper.prototype.getDockerImageName = function () {
+    if(this._aspNetVersion === '1.0.0-rc1-final') {
+        return 'microsoft/aspnet:1.0.0-rc1-update1';
```

Comment:
Can you only add a quick comment here on why we check for `rc1-final`, but return `rc1-update1`? Might make total sense now, but later on someone will probably try to figure out what was going on :)